page-factory-api/src/main/java/ru/sbtqa/tag/pagefactory/environment/Environment.java

```diff
@@ -0,0 +1,39 @@
+package ru.sbtqa.tag.pagefactory.environment;
+
+import ru.sbtqa.tag.pagefactory.actions.PageActions;
+import ru.sbtqa.tag.pagefactory.checks.PageChecks;
+import ru.sbtqa.tag.pagefactory.drivers.DriverService;
+
+/**
+ * Static storage for infrastructure
+ */
+public class Environment {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)